tracer/build/_build/_build.csproj

```diff
@@ -2,7 +2,9 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework Condition="$(OS.StartsWith('Windows'))">$(TargetFramework)-windows10.0.17763</TargetFramework>
```

Comment:
what is this line 6 for?

Reply:
The nuget `Microsoft.Toolkit.Uwp.Notifications` contains a build that's specific to Windows that contains the code to show the notification. If you don't have this line, msbuild will fall back on the generic version, so the `Show()` method will be mising.